src/Ombi/ClientApp/app/requests/movierequests.component.html

```diff
@@ -14,32 +14,32 @@
+                                <a class="active" (click)="setOrder('requestedDate', $event)">{{ 'Requests.SortRequestDate' | translate }}
                                         <span *ngIf="order === 'requestedDate'">
                                         <span [hidden]="reverse"><small><i class="fa fa-arrow-down" aria-hidden="true"></i></small></span>
                                         <span [hidden]="!reverse"><small><i class="fa fa-arrow-up" aria-hidden="true"></i></small></span>
                                         </span>
                                     </a>
-                            <a (click)="setOrder('title')">{{ 'Requests.SortTitle' | translate}}
+                            <a (click)="setOrder('title', $event)">{{ 'Requests.SortTitle' | translate}}
                                     <span *ngIf="order === 'title'">
                                             <span [hidden]="reverse"><small><i class="fa fa-arrow-down" aria-hidden="true"></i></small></span>
                                             <span [hidden]="!reverse"><small><i class="fa fa-arrow-up" aria-hidden="true"></i></small></span>
                                             </span>
                             </a>                       
-                            <a (click)="setOrder('releaseDate')">{{ 'Requests.TheatricalReleaseSort' | translate }}
+                            <a (click)="setOrder('releaseDate', $event)">{{ 'Requests.TheatricalReleaseSort' | translate }}
                                     <span *ngIf="order === 'releaseDate'">
                                             <span [hidden]="reverse"><small><i class="fa fa-arrow-down" aria-hidden="true"></i></small></span>
                                             <span [hidden]="!reverse"><small><i class="fa fa-arrow-up" aria-hidden="true"></i></small></span>
                                             </span>
                             </a>                      
-                            <a (click)="setOrder('requestedUser.userAlias')">{{ 'Requests.SortRequestedBy' | translate }}
+                            <a (click)="setOrder('requestedUser.userAlias', $event)">{{ 'Requests.SortRequestedBy' | translate }}
                                     <span *ngIf="order === 'requestedUser.userAlias'">
                                             <span [hidden]="reverse"><small><i class="fa fa-arrow-down" aria-hidden="true"></i></small></span>
                                             <span [hidden]="!reverse"><small><i class="fa fa-arrow-up" aria-hidden="true"></i></small></span>
                                             </span>
                                         </a>
 
-                            <a (click)="setOrder('status')">{{ 'Requests.SortStatus' | translate }}
+                            <a (click)="setOrder('status', $event)">{{ 'Requests.SortStatus' | translate }}
```

Comment:
Passing the click event to the setOrder function so that we can use it to do active style CSS  class toggling based on the currently selected sort filter.